cli/src/commands/extensions/permissions.rs

```diff
@@ -276,9 +244,24 @@ impl From<&Permissions> for PermissionsOptions {
 
 /// Construct sandbox with a set of pre-defined acceptable exceptions.
 #[cfg(unix)]
-pub fn default_sandbox() -> SandboxResult<Birdcage> {
+pub fn default_sandbox(strict: bool, min_landlock_abi: u8) -> SandboxResult<Birdcage> {
+    #[cfg(not(target_os = "linux"))]
     let mut birdcage = Birdcage::new()?;
 
+    #[cfg(target_os = "linux")]
+    let abi = match min_landlock_abi {
+        3 => LANDLOCK_ABI::V3,
+        2 => LANDLOCK_ABI::V2,
+        _ => LANDLOCK_ABI::V1,
+    };
+    #[cfg(target_os = "linux")]
+    let mut birdcage = Birdcage::new_with_version(abi)?;
+
+    // Do not add any default exceptions for strict sandboxes.
+    if strict {
+        return Ok(birdcage);
+    }
```

Comment:
Moved this inside the function since I think it's better to handle it all in one place.